engine/physics/public/physics_module.hpp

```diff
@@ -211,13 +230,19 @@ class DebugRendererSimpleImpl : public JPH::DebugRendererSimple
         return linePositions;
     }
 
+    [[nodiscard]] const std::vector<glm::vec3>& GetPersistentLinesData() const
```

Comment:
NO_DISCARD

Reply:
have we actually decided on this? there are places where [[nodiscard]] is still being used

Reply:
If i remember coorectly we have, those cases should be resolved then when we get to it as well.